### ISSUE TITLE
deploy artifacts in maven central

### ISSUE THREAD

**Issue by jimkont** [RMLio/RML-Mapper#16]
Would be very cinvenient to deploy all artifacts in maven central.

**Comment by m1ci** [comment]
yup, I agree, we are also using RML and having RML in maven central would be great!
